tests/common/test_time.py

```diff
@@ -100,3 +102,13 @@ def test_ensure_pendulum_date_utc() -> None:
     assert ensure_pendulum_date(
         datetime(2021, 1, 1, 0, 0, 0, tzinfo=timezone(timedelta(hours=8)))
     ) == pendulum.date(2020, 12, 31)
+
+
+@pytest.mark.parametrize("date_value,datetime_obj", test_params)
+def test_datetime_to_timestamp_helpers(
+    date_value: TAnyDateTime, datetime_obj: pendulum.DateTime
+) -> None:
+    dt = ensure_pendulum_datetime(date_value)
+    assert isinstance(dt, pendulum.DateTime)
+    assert datetime_to_timestamp(datetime_obj) == int(datetime_obj.timestamp())
```

Comment:
what you want to do here is to create a date time object with a value that you select and then compare it to an actual integer that it should produce, this way you are basically just testing the same code again:
datetime(2021, 1, 1, 5, 6, 55) == 123456789

Reply:
Nice, I will adjust this with parametrized fixture